### ISSUE TITLE
Shortcut assigned to color calibration module - show module is not working

### ISSUE THREAD

**Issue by vancouverbluesea** [darktable-org/darktable#8065]
**Describe the bug**

I would like to have a shortcut to color calibration module - "show module" specifically. However - when I am to assign a shortcut to it and try to use it - nothing happens. The behaviour is the same regardless of key combination chosen and the same key combination would work well when used in different area - for example color calibration module "enable module"

**To Reproduce**

1. Go to settings - shortcuts
2. Navigate to calibration module and try to assign a short cut
3. After the shortcut is assigned - go to darkroom and try to use it.

**Screenshots**

![image](https://user-images.githubusercontent.com/69771189/106529015-9c033b80-649e-11eb-88da-28b27772d83d.png)

**Expected behavior**

Should display the module so the user can interact with it

**Screenshots**
![image](https://user-images.githubusercontent.com/69771189/106529453-54c97a80-649f-11eb-9a92-9c17024593ea.png)

**Actual behavior**
Nothing happens

**Platform (please complete the following information):**
 - Darktable Version:  3.4.0
 - OS:  Linux PopOS 20.04
 - OpenCL yes - active
 -  Nvidia RTX 2070 mobile

**Additional context**

**Comment by elstoc** [comment]
I've tested this on both 3.4 and master (3.5.0+897~g88067a4e8) and am unable to reproduce the issue.

**Comment by vancouverbluesea** [comment]
@elstoc 
I think I figured out what is happening. If the module **is not** added to any group (even favourites). Then the shortcut does not work. However - if the module is added to at least one group (in my case favourites) - the shortcut works as expected.
![2021-02-01_18-02](https://user-images.githubusercontent.com/69771189/106542041-ecd35e00-64b7-11eb-9f86-15975adad6c1.png)

To make things a bit more complicated - I did indicate initially that I have selected the on/off (the currently active) module group. But in reality - I am not always on this group. So - when I am selecting the active modules group - the shortcut does work (even if it is not in another group as favourites) - but when I navigate away from that particular group the shortcut stops working. 

As an opposite - if the shortcut is part of favourites - even if I do not select the favourites group - executing the shortcut does bring me to the module.

Currently I am not sure if this is expected behaviour or unexpected. Unless somebody thinks otherwise - I will consider it _expected_ and will close the case.
